docs/platforms/javascript/common/configuration/options.mdx

```diff
@@ -63,6 +63,12 @@ Sets the URL that will be used to transport captured events. This can be used to
 
 </SdkOption>
 
+<SdkOption name="sendDefaultPii" type='boolean' defaultValue='false'>
+
+Set this option to `true` to send default PII data to Sentry. Amongst other things, enabling this will enable automatic IP address collection on events.
```

Comment:
Apparently, "amongst" is more common in british english and also more formal. 
```suggestion
Set this option to `true` to send default PII data to Sentry. Among other things, enabling this will enable automatic IP address collection on events.
```